### PR TITLE
docs: Rename CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Rename the contributing guide to make https://www.inspektor-gadget.io/docs/latest/devel/contributing/ work again. The change is needed to fix the contribute link on the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Inspektor Gadget directly. End-users do not need this and can use
 
 ## Contributing
 
-Contributions are welcome, see [CONTRIBUTING](docs/devel/CONTRIBUTING.md).
+Contributions are welcome, see [CONTRIBUTING](docs/devel/contributing.md).
```
Comment (mauriciovasquezbernal) [inline README.md:174]: I'll point it to the website to avoid too many redirects.

**Review by mauriciovasquezbernal — APPROVED** [review]
Once minor nit. LGTM otherwise.

File: README.md (hunk 1 above)
Comment (mauriciovasquezbernal) [reply README.md:174]: mmm, it's actually pointing to the right file, nothing to do here, sorry for the noise.

**Comment by mqasimsarfraz** [comment]
Thanks for the review!

**Pull request merged**